src/qibolab/instruments/abstract.py

```diff
@@ -2,7 +2,7 @@
 from abc import ABC, abstractmethod
 from pathlib import Path
 
-from qibolab.paths import user_folder
+INSTRUMENTS_DATA_FOLDER = Path.home() / ".qibolab" / "instruments" / "data"
```

Comment:
Do we really need instruments data?

If yes, we need to think about a reliable solution, because retrieving information from the file system is not always trivial (privileges, different file systems, ...).

It would be helpful to know where this information is actually consumed (I suppose in the actual instrument, so it should be available when you load the platform).

Reply:
Regarding this I remember that there was a discussion a few months ago with @aorgazf...
I don't think that they should be needed anymore